llvm/lib/Transforms/IPO/FunctionSpecialization.cpp

```diff
@@ -809,20 +908,40 @@ bool FunctionSpecializer::findSpecializations(Function *F, unsigned FuncSize,
       auto IsProfitable = [](Bonus &B, unsigned Score, unsigned FuncSize,
                              unsigned FuncGrowth) -> bool {
         // No check required.
-        if (ForceSpecialization)
+        if (ForceSpecialization) {
+          LLVM_DEBUG(dbgs() << "FnSpecialization: Force is on\n");
           return true;
+        }
         // Minimum inlining bonus.
-        if (Score > MinInliningBonus * FuncSize / 100)
+        if (Score > MinInliningBonus * FuncSize / 100) {
+          LLVM_DEBUG(dbgs()
+                     << "FnSpecialization: Sufficient inlining bonus( " << Score
```

Comment:
space followed by parenthesis, not the opposite
"FnSpecialization: Sufficient inlining bonus( " -> "FnSpecialization: Sufficient inlining bonus ("